synapse/lib/link.py

```diff
@@ -15,7 +15,8 @@
 import synapse.lib.const as s_const
 import synapse.lib.msgpack as s_msgpack
 
-readsize = 10 * s_const.megabyte
+READSIZE = 10 * s_const.megabyte
```

Comment:
The `READSIZE` changes are just for consistent use of module local vars being all upper case :)

Reply:
Thank you :D